### PR TITLE
Adding functional test for CacheTagHelper's enabled property.

### PR DESCRIPTION
**Pull request by sornaks**

### CODE REVIEW

File: test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTest.cs [hunk 1]
```diff
@@ -383,6 +383,25 @@ public class MvcTagHelpersTest
...
+            client.BaseAddress = new Uri("http://localhost");
+
+            // Act
+            var response1 = await client.GetStringAsync("/catalog/GetDealPercentage/20?isEnabled=true");
+            var response2 = await client.GetStringAsync("/catalog/GetDealPercentage/40?isEnabled=true");
+            var response3 = await client.GetStringAsync("/catalog/GetDealPercentage/30?isEnabled=false");
+
+            // Assert
+            Assert.Equal("Deal percentage is 20", response1.Trim());
+            Assert.Equal("Deal percentage is 20", response2.Trim());
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTagHelpersTest.cs]: `Assert.Equal(response1, response2)`

**Comment by NTaylorMullen** [comment]
:shipit:

**Comment by dougbu** [comment]
:shipit: once you figure out why the CIs are failing (looks like Coherence-Signed and therefore the dev feed hasn't caught up)

**Comment by sornaks** [comment]
Thank you! https://github.com/aspnet/Mvc/commit/7ea467f73dd6d6e166284e5fcf6a1ed40efbadcb

**Pull request closed without merging**